sdk/storage/storage-common/src/PooledBuffer.ts

```diff
@@ -81,7 +81,7 @@ export class PooledBuffer {
   constructor(capacity: number, buffers: Buffer[], totalLength: number);
   constructor(capacity: number, buffers?: Buffer[], totalLength?: number) {
     this.capacity = capacity;
-    this._size = capacity;
+    this._size = 0;
 
```

Comment:
Is this another bug? Does this cause other data integrity issues?

Reply:
It won't be triggered as we always call fill() afterwards.